libs/model/test/subscription/thread-subscription-lifecycle.spec.ts

```diff
@@ -89,10 +89,21 @@ describe('Thread subscription lifecycle', () => {
         { user_id: actor.user.id!, thread_id: threadTwo!.id! },
       ]);
 
+    expect(threadSubOne).to.exist;
+    expect(threadSubTwo).to.exist;
+
     const res = await query(GetThreadSubscriptions(), {
       actor,
       payload: {},
     });
+
+    expect(res).to.exist;
+    expect(res![0]).to.exist;
+    expect(res![1]).to.exist;
+
+    res![0]!.Thread = null!;
```

Comment:
Why are we modifying the response from the API and then testing it? Shouldn't we add the thread to the deep members check below instead?

Reply:
It's a good question. The resulting types are different.

I'm going to rework this and then we can test them properly:

https://github.com/hicommonwealth/commonwealth/issues/8243

The issue is that the types are different from the real thread vs what we're getting back from the API so the equals will not work.  

I'm not even sure how this test worked to begin with.

Reply:
@timolegros I"m going to resolve this but fix it in another PR.